staging/src/k8s.io/apiserver/pkg/server/filters/wrap.go

```diff
@@ -51,16 +50,15 @@ func WithPanicRecovery(handler http.Handler, resolver request.RequestInfoResolve
 			// This call can have different handlers, but the default chain rate limits. Call it after the metrics are updated
 			// in case the rate limit delays it.  If you outrun the rate for this one timed out requests, something has gone
 			// seriously wrong with your server, but generally having a logging signal for timeouts is useful.
-			runtime.HandleError(fmt.Errorf("timeout or abort while handling: %v %q", req.Method, req.URL.Path))
+			runtime.HandleError(fmt.Errorf("timeout or abort while handling: %s", request.GetVerbURIAndAuditIDString(req)))
```

Comment:
What do we buy with this untyped way of writing this?

Reply:
having a consistent key names (with structured logging). But I remove it and doing it inline now.